osgp/shared/osgp-dto/src/main/java/org/opensmartgridplatform/dto/valueobjects/smartmetering/PowerQualityObjectDto.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2017 Smart Society Services B.V.
```

Comment:
This file looks like it is newly added, constructor should mention the current year and Alliander N.V. in these cases.

Reply:
Changed